File: week02/ee/농구경기.py
import sys
sys.stdin = open("input.txt")

N = int(input())
arr = [0] * 27

for _ in range(N):
    arr[ord(input()[0])-96] += 1

answer = []
for i in range(1, 27):
    if arr[i] >= 5:
        answer.append(chr(i+96))

if len(answer):
    answer.sort()
    print("".join(answer))
else:
    print("PREDAJA")

File: week02/ee/농구경기.js
// 농구경기

const fs = require("fs");
const filePath = process.platform === "linux" ? "dev/stdin" : "./input.txt";

const [N, ...arr] = fs.readFileSync(filePath).toString().trim().split("\n");

// console.log("a".charCodeAt(0)) // 97

let cntArr = Array.from({ length: 26 }, () => 0)
for (let i = 0; i < N; i++) {
  cntArr[arr[i][0].charCodeAt(0) - 97] += 1
}

let answer = [];
for (let k = 0; k < 26; k++) {
  if (cntArr[k] >= 5) answer.push(String.fromCharCode(k + 97));
}

console.log(answer.length ? answer.sort().join("") : "PREDAJA")